pkg/metrics/metrics.go

```diff
@@ -0,0 +1,141 @@
+package metrics
+
+import (
+	logging "github.com/op/go-logging"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	subsystem = "asb"
+)
+
+var (
+	sandboxCreated = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Subsystem: subsystem,
+			Name:      "sandbox_created",
+			Help:      "Counter of all sandbox namespaces that are created.",
+		})
+
+	sandboxDeleted = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Subsystem: subsystem,
+			Name:      "sandbox_deleted",
+			Help:      "Counter of all sandbox namespaces that are deleted.",
+		})
+
+	specsLoaded = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Subsystem: subsystem,
+			Name:      "specs_loaded",
+			Help:      "Specs loaded from registries, partitioned by registry name.",
+		}, []string{"registry_name"})
+
+	specsReset = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Subsystem: subsystem,
+			Name:      "specs_reset",
+			Help:      "Counter of how many times the specs have been reset.",
+		})
+
+	provisionJob = prometheus.NewGauge(
```

Comment:
I think another good metric is a counter of the number of things provisioned and deprovisioned. Ditto bind and unbind.

Reply:
Can you add to https://docs.google.com/document/d/1ui57sb3kMf2HEt6LelfuDEGjrAhbSaD4NhYPljecyRE/edit# 

I think that we can add better metrics for 3.8, but need to get some basics up today. 

Do you feel super strongly that we should have them for the initial PR?

Reply:
Makes sense to me to get a baseline and then iterate once we have some feedback on what is useful in the wild.

Reply:
IMO keeping track of number of bind/unbind is important. It's one the four APIs we track and care about. We can add better metrics for 3.8, but to me this seems like a core metric we want.